perllib/FixMyStreet/Cobrand/CheshireEast.pm

```diff
@@ -8,11 +20,71 @@ use Moo;
 with 'FixMyStreet::Roles::ConfirmOpen311';
 with 'FixMyStreet::Roles::ConfirmValidation';
 
+=head2 Defaults
+
+=over 4
+
+=cut
+
 sub council_area_id { 21069 }
 sub council_area { 'Cheshire East' }
 sub council_name { 'Cheshire East Council' }
 sub council_url { 'cheshireeast' }
 
+=item * We restrict use of the contact form to abuse reports only.
+
+=cut
+
+sub abuse_reports_only { 1 }
+
+=item * Users with a cheshireeast.gov.uk email can always be found in the admin.
+
+=cut
+
+sub admin_user_domain { 'cheshireeast.gov.uk' }
+
+=item * The default map zoom is set to 3.
+
+=cut
+
+sub default_map_zoom { 3 }
+
+=item * Fetched report description is not shown.
+
+=cut
+
+sub filter_report_description { "" }
+
+=item * Uses the OSM geocoder.
+
+=cut
+
+sub get_geocoder { 'OSM' }
+
+=item * Map type is CheshireEast.
```

Comment:
I'd put this one up one before  the map zoom as they are very linked (the zoom default is different because it's a totally different map type).